src/gentropy/method/susie_inf.py

```diff
@@ -469,43 +470,55 @@ def cred_inf(
     @staticmethod
     def credible_set_qc(
         cred_sets: StudyLocus,
-        study_index: StudyIndex,
-        ld_index: LDIndex,
         p_value_threshold: float = 1e-5,
         purity_min_r2: float = 0.01,
+        clump: bool = False,
+        ld_index: LDIndex | None = None,
+        study_index: StudyIndex | None = None,
         ld_min_r2: float = 0.8,
     ) -> StudyLocus:
         """Filter credible sets by lead P-value and min-R2 purity, and performs LD clumping.
```

Comment:
```suggestion
        """Filter credible sets by lead P-value and min-R2 purity, and performs LD clumping.
        In case of duplicated loci, the filtering retains the loci wth the lowest credibleSetLog10BF
```